homeassistant/components/enphase_envoy/number.py

```diff
@@ -25,6 +25,10 @@
 from .coordinator import EnphaseConfigEntry, EnphaseUpdateCoordinator
 from .entity import EnvoyBaseEntity
 
+# coordinator is used to get all data centralized.
```

Comment:
```suggestion
```
this part actually only applies to read-only platforms (sensor & binary sensor), but I think you can also get rid of the second comment line as it adds no value imo

Reply:
Not sure if I get it. All platform data for this integration is refreshed through the coordinator _async_update_data. When a number is changed then coordinator is requested to refresh as well by using self.coordinator.async_request_refresh() which will collect all platform data agaian. So I would think the comment still makes sense.

As for the second comment I was thinking that it will prevent parrallel data sends if some automaion changes multiple in one go. Although the debouncer might assist with that too.

Reply:
I would agree the comment lines can go in all the files as they don't provide additional value.

Reply:
Ok, done, thanks for the review and feedback.

Reply:
The comment is true, but non-read platforms send their data through actions so the constant applies there. On sensor and binary sensor the constant has no effect with a coordinator, so you can just set it to 0.

Reply:
Thanks for the explanation. I see that the latest quality doc version has that same note as well. I'll update the sensor and binary-sensor.